tests/charts/nm_vllm_grafana_dashboards/test_config_map_list.py

```diff
@@ -0,0 +1,59 @@
+from typing import Dict, Optional
+
+from pytest_helm_templates import HelmRunner
+
+
+def test_expected_dashboard_config_maps_are_included(
```

Comment:
@DaltheCow can you add another test to validate that `label` and `labelValue` are configurable, please?

Reply:
Sure!